src/prompts/views.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from api import settings as api_settings
+from api.utils import failure_response
+from api.utils import success_response
+from prompts.models import Prompt
+from rest_framework import generics
+from rest_framework import status
+
+from .controllers.create_prompt_controller import CreatePromptController
+# from .controllers.update_prompt_controller import UpdatePromptController
```

Comment:
delete comments

Reply:
remember to reinstall the pre commit hooks on your new laptop so that if formats everything nicely for you before committing !